x-pack/legacy/plugins/fleet/public/lib/agent.ts

```diff
@@ -14,12 +14,28 @@ export class AgentsLib {
     private readonly elasticsearch: ElasticsearchLib
   ) {}
 
-  /** Get a single beat using it's ID for lookup */
+  /**
+   * Get an agent by id
+   * @param id
+   */
   public async get(id: string): Promise<Agent | null> {
     const agent = await this.adapter.get(id);
     return agent;
   }
 
+  /**
+   * Get an agent by id
+   * @param id
+   */
+  public async getAgentEvents(
+    id: string,
+    search: string,
+    page: number,
```

Comment:
yay we used the [same param names](https://github.com/elastic/kibana/pull/47621/files#diff-05beb603c5e81930e05afa0f9e71fcd0R37) for paging!